locales/de.json

```diff
@@ -20,17 +20,17 @@
-        "JavaScript":"Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quaerat animi voluptatum aut commodi excepturi ut nihil illo, aspernatur nesciunt numquam iste quae eaque ipsam in velit repellendus pariatur quam quo.",
-        "jQuery":"Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quaerat animi voluptatum aut commodi excepturi ut nihil illo, aspernatur nesciunt numquam iste quae eaque ipsam in velit repellendus pariatur quam quo.",
-        "sass":"Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quaerat animi voluptatum aut commodi excepturi ut nihil illo, aspernatur nesciunt numquam iste quae eaque ipsam in velit repellendus pariatur quam quo.",
-        "gulp":"Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quaerat animi voluptatum aut commodi excepturi ut nihil illo, aspernatur nesciunt numquam iste quae eaque ipsam in velit repellendus pariatur quam quo.",
-        "git":"Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quaerat animi voluptatum aut commodi excepturi ut nihil illo, aspernatur nesciunt numquam iste quae eaque ipsam in velit repellendus pariatur quam quo.",
-        "rwd":"Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quaerat animi voluptatum aut commodi excepturi ut nihil illo, aspernatur nesciunt numquam iste quae eaque ipsam in velit repellendus pariatur quam quo.",
-        "react":"Lorem ipsum dolor sit amet, consectetur adipisicing elit. Reiciendis omnis sequi, asperiores, officia a aliquid obcaecati consectetur, tempore illum veniam odio architecto maiores minima magni voluptas similique vel eius soluta?",
-        "ajax":"Lorem ipsum dolor sit amet, consectetur adipisicing elit. Labore, quia voluptas? Hic ut quae, ex enim quo, porro eum itaque earum, non commodi odio praesentium quos vel iste nihil nemo.",
-        "wordpress":"Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quaerat animi voluptatum aut commodi excepturi ut nihil illo, aspernatur nesciunt numquam iste quae eaque ipsam in velit repellendus pariatur quam quo."
+        "html5": "Durch HTML beschreibe ich und definiere die Inhalte und Struktur der Websiten, dazu noch Grundlagen von Layout; ich füge Hyperlinks, Multimedia, Formulare ein.",
+        "css3":"Durch CSS mache ich eine visuelle Darstellung der Website, definiere Layout, Farben, Typografie und mache einfache Animationen.",
+        "JavaScript":"Es ist eine Programmiersprache mit welcher  ich die Website interaktiviere, sie reagiert auf  Events(z.B. durch Mouseclick), sie gesteltet  Veränderungen, Animationen und Effekte.",
+        "jQuery":"Eine Bibliothek von JavaScript welche vereinfacht das Programmieren. Manchmal anstatt viele Linien von Code in JS zu schreiben, brauchen wir nur eine Linie in jQuery.",
+        "sass":"Eine Skriptsprache, deren Code in Ergebnisdateien verarbeitet wird CSS. Dies hilft, CSS viel besser organisiert zu sein, ist leicht zu erstellen; außerdem ist Code besser lesbar.",
```

Comment:
Eine Skriptsprache, deren Code in Ergebnisdateien verarbeitet wird CSS. Dies hilft, CSS besser zu organisieren, ist leicht zu erstellen und der Code ist besser lesbar.